src/main/resources/config/application-dev.yml

```diff
@@ -72,64 +93,38 @@ spring:
     kafka:
         bootstrap-servers: localhost:9093
 
-liquibase:
-    contexts: dev
-    enabled: true
-
-# ===================================================================
-# To enable SSL, generate a certificate using:
-# keytool -genkey -alias uaa -storetype PKCS12 -keyalg RSA -keysize 2048 -keystore keystore.p12 -validity 3650
-#
-# You can also use Let's Encrypt:
-# https://maximilian-boehm.com/hp2121/Create-a-Java-Keystore-JKS-from-Let-s-Encrypt-Certificates.htm
-#
-# Then, modify the server.ssl properties so your "server" configuration looks like:
-#
-# server:
-#    port: 8443
-#    ssl:
-#        key-store: keystore.p12
-#        key-store-password: <your-password>
-#        keyStoreType: PKCS12
-#        keyAlias: uaa
-# ===================================================================
 server:
     port: 9999
 
 # ===================================================================
 # JHipster specific properties
 #
-# Full reference is available at: https://jhipster.github.io/common-application-properties/
+# Full reference is available at: https://www.jhipster.tech/common-application-properties/
 # ===================================================================
 
 jhipster:
     http:
-        version: V_1_1 # To use HTTP/2 you will need SSL support (see above the "server.ssl" configuration)
+        version: V_1_1 # To use HTTP/2 you will need to activate TLS (see application-tls.yml)
     # CORS is only enabled by default with the "dev" profile, so BrowserSync can access the API
     cors:
         allowed-origins: "*"
-        allowed-methods: GET, PUT, POST, DELETE, OPTIONS
+        allowed-methods: "*"
         allowed-headers: "*"
-        exposed-headers:
+        exposed-headers: "Authorization,Link,X-Total-Count"
         allow-credentials: true
         max-age: 1800
     security:
         client-authorization:
-            client-id:
-            client-secret:
+            client-id: internal
+            client-secret: internal
     mail: # specific JHipster mail property, for standard properties see MailProperties
-        from: <tenantname>@xm-online.com
-        base-url: http://127.0.0.1:9999
+        from: uaa@localhost
```

Comment:
As far as i know there is some dependency in mail sending code which should use <tenantname> placeholder. So please check if it is not break the functionality.

Reply:
done